### PR TITLE
Add primalize benchmark

### PR DESCRIPTION
**Pull request by jgaskins**
@mediafinger asked if I could push my branch where I added Primalize to the benchmark. Nice work making it so easy to extend!

Primalize is a type-checking serializer library, written while I was at a previous company to validate their API output. I wrote a [blog post](https://dev.to/jgaskins/fast-type-checked-serializers-for-ruby-web-apis-2o19) about the type-checking part of it, but we also discovered at the time that it was pretty fast, so I figured when I saw Alba in Ruby Weekly and there was a benchmark already written that I'd check it against Primalize. 🙂

### CODE REVIEW

File: benchmark/local.rb [hunk 1]
```diff
@@ -111,6 +112,28 @@ def commenter_names
   end
 end
 
+# --- Primalize serializers ---
```
Comment (mediafinger) [inline benchmark/local.rb:115]: Could you move that down, just above the line:

`# --- Representable serializers ---`

to keep this alphabetically sorted as well?

**Comment by okuramasafumi** [comment]
@jgaskins hi, thank you for your Pull Request! I'm going to merge this PR soon after you amend you commit to keep alphabetical order.

**Comment by jgaskins** [comment]
@okuramasafumi Thanks for the reminder, this completely slipped my mind.

tfw you forget to respond to a notification

**Comment by okuramasafumi** [comment]
@jgaskins Could you squash your last commit into the first one?

**Comment by jgaskins** [comment]
@okuramasafumi Is there another reason you need me to do this manually? GitHub's squash-merge option should obviate the need for manual squashing.

**Comment by okuramasafumi** [comment]
@jgaskins "squash-merge" was disabled for some reason, but now that I enabled it, I can squash-merge this PR. Thanks!

**Pull request merged**